tools/README.md

```diff
@@ -100,12 +100,16 @@ We outline below the rapid startup steps to get SonarQube Server setup. Refer to
 
 You will need to use the `sonarqube-postgresql-template.yaml` file in order to deploy it to openshift. This will deploy BOTH the PostgreSQL database AND the SonarQube server.
 
-*Note: At the time of writing, the master branch of BCDevOps SonarQube repo was at commit `bbb9f62e29706b61382cf24d7ad7e08f2476a01f`.  And the template was at commit `bc80961d75eed66ec70ca022a6444963341fb39f`.*
+*Note: At the time of writing, the master branch of BCDevOps SonarQube repo was at commit `d4d0cf1b4237588058c2d1e9302c6fb30acd5752` and we are using the template found at commit `d8b63af664ec2e88026e13885e6acd85d8763c79`.*
 
 Deploying the database is done with the following:
 
 ```sh
-oc -n $tools new-app -f https://raw.githubusercontent.com/BCDevOps/sonarqube/bc80961d75eed66ec70ca022a6444963341fb39f/sonarqube-postgresql-template.yaml --param=SONARQUBE_VERSION=6.7.5
+export sq_commit=d8b63af664ec2e88026e13885e6acd85d8763c79
+
+oc project $tools
+
+oc -n $tools new-app -f https://raw.githubusercontent.com/BCDevOps/sonarqube/$sq_commit/sonarqube-postgresql-template.yaml --param=SONARQUBE_VERSION=6.7.7
```

Comment:
why wouldn't you parameterize the version along with the commit hash?

Reply:
The commit hash (not in the template) answers the question "which version of the openshift template should I source from?", while the version parameterization (which is in the template) answers the question "which bcgov tagged image should I deploy?". The commit hash is external of the template that is used - the hash version and the the image version don't originate from the same commit either as there were some other bugs with the attempted update.

There were some further changes to the template I didn't agree with for more recent releases (specifically the forced login by default), but to undo that, we'd have to copy their template into this repository. This is the best short-term compromise I could come up with to ensure we're using an upstream template (locked to a version) but still update the image used.